src/main/java/de/tum/in/www1/artemis/service/connectors/GitService.java

```diff
@@ -882,8 +916,6 @@ private void setRemoteUrl(Repository repo) {
      */
     public void pullIgnoreConflicts(Repository repo) {
         try (Git git = new Git(repo)) {
-            // flush cache of files
-            repo.setContent(null);
```

Comment:
Why is this not needed anymore?

Reply:
We removed this caching mechanism to prevent outdated data from causing problems:

[src/main/java/de/tum/in/www1/artemis/domain/Repository.java](https://github.com/ls1intum/Artemis/pull/8546/files#diff-7124501c6ce3e5419ce1ee6540a58132f6d0a1bc99f7f1856abc15d7166c324e)